lms/static/sass/course/courseware/_courseware_search.scss

```diff
@@ -61,7 +62,7 @@
 
   .search-results-item {
     position: relative;
-    border-bottom: 1px solid lighten($border-color, 10%);
+    border-bottom: 1px solid $border-color-1;
```

Comment:
Sorry I didn't notice this earlier. I don't know that this is a variable we'd like to continue using, I think this should be changed to use one of our basic color variables. I've not seen us use $border-color before though, where is it being referenced from?